.github/workflows/devskim.yml

```diff
@@ -0,0 +1,31 @@
+name: DevSkim
```

Comment:
do we really want to pick devskim for the release branch?

Reply:
Ah, good catch, I accidentally cherry-picked that as well. I think it should be OK, though, since it only affects GH workflows and not Azure Pipelines; happy to open a new PR with that fixed, though, if you prefer.